### ISSUE TITLE
[Suggestion] Add example of excluding cold/frozen data from Indicator Match rule queries

### ISSUE THREAD

**Issue by rylnd** [elastic/security-docs#5883]
### What can we change to make the docs better?

What: we should include an example of how to exclude cold and/or frozen tier data from rule queries.
Why: we mention that Indicator Match rules do not support cold/frozen tier data, but by default those tiers will still be searched if present. This is the source of many support requests, as well as several SDHs. It should be included in our documentation as a best practice.

### Doc URL

Please include the doc URL and any other related information where applicable:
Doc URL: https://www.elastic.co/guide/en/security/current/detection-engine-overview.html#support-indicator-rules
Github issue link(s)/Other resources: https://github.com/elastic/sdh-security-team/issues/1055

### Which documentation set needs improvement?

ESS and serverless

### Software version

Applies to all kibana versions with Indicator Match rules.

**Comment by vitaliidm** [comment]
There was already an effort to document how to filter out data tiers in rules
https://github.com/elastic/security-docs/issues/5483
https://github.com/elastic/security-docs/pull/5849/files

Maybe that can be helpful

**Comment by nastasha-solomon** [comment]
@rylnd would [these instructions](https://www.elastic.co/guide/en/security/8.16/exclude-cold-frozen-data-individual-rules.html) work for excluding cold and frozen data from indicator match rules? Or are these docs not fully applicable because indicator match rules require more targeted filtering  (e.g., users must filter out cold/frozen data from the rule's threat index and target index)?

**Comment by rylnd** [comment]
@nastasha-solomon yeah, that should be enough to allow a rule author to add those filters to the rule. Maybe we can just link to that page from the one I referenced?
